environment.yml

```diff
@@ -29,23 +29,21 @@ dependencies:
 - pyface>=6
 - traitsui>=6
 - imageio>=2.6.1
+- imageio-ffmpeg>=0.4.1
+- vtk
+- pyvista>=0.24
+- nibabel
+- nilearn
+- pytest-sugar
+- pydocstyle
+- codespell
+- python-picard
+- tqdm
+- python-neo
+- dipy
+- pysurfer
 - pip:
   - mne
   - https://github.com/numpy/numpydoc/archive/master.zip
```

Comment:
is there a reason to still use numpydoc master now that they have a 1.0 release?

Reply:
I have no idea. @larsoner?